vochain/indexer/process.go

```diff
@@ -394,13 +402,22 @@ func (s *Indexer) setResultsHeight(pid []byte, height uint32) error {
 	if height == 0 {
 		panic("setting results height to 0?")
 	}
-	queries, ctx, cancel := s.timeoutQueries()
+
+	ctx := context.TODO()
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
-	if _, err := queries.SetProcessResultsHeight(ctx, indexerdb.SetProcessResultsHeightParams{
-		ID:            pid,
-		ResultsHeight: int64(height),
+
+	if err := s.ExecTx(ctx, func(q *indexerdb.Queries) error {
+		if _, err := q.SetProcessResultsHeight(ctx, indexerdb.SetProcessResultsHeightParams{
+			ID:            pid,
+			ResultsHeight: int64(height),
+		}); err != nil {
+			return err
+		}
+		return nil
```

Comment:
does this change help in any way?

there is no need to modify the other queries; I would focus on just newProcessPool. the only query to update is newEmptyProcess.